recipe/meta.yaml

```
@@ -1,29 +1,30 @@
 {% set name = "wfdb" %}
 {% set version = "4.1.2" %}
-{% set sha256 = "6acef3ab2759f60cf911a57d9e5214ea15bd17b5ad4b3c66e4a57a7690e16024" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/MIT-LCP/{{ name }}-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c96c62b8abede8c0987c77d49285c923f114787a96d110cf95dac5e5d56360cd
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
+  # pysound not available for s390x or win
+  # No poetry for python 3.12
+  skip: true  # [py<37 or py>311 or s390x or win]
 
 requirements:
   host:
     - pip
     - poetry
```

```suggestion
    - poetry-core
```
https://github.com/MIT-LCP/wfdb-python/blob/v4.1.2/pyproject.toml#L36